lib/std/os/linux/io_uring.zig

```diff
@@ -636,6 +636,51 @@ pub const IO_Uring = struct {
             @ptrCast(*const c_void, fds.ptr),
             @intCast(u32, fds.len),
         );
+        try convert_registration_error(res);
+    }
+
+    /// Registers the file descriptor for an eventfd that will be notified of completion events on
+    //  an io_uring instance.
+    pub fn register_eventfd(self: *IO_Uring, fd: os.fd_t) !void {
+        assert(self.fd >= 0);
+        comptime assert(@sizeOf(os.fd_t) == @sizeOf(c_int));
+        const res = linux.io_uring_register(
+            self.fd,
+            .REGISTER_EVENTFD,
+            @ptrCast(*const c_void, &fd),
+            @intCast(u32, 1),
```

Comment:
This can be demoted to [`@as(u32, 1)`](https://ziglang.org/documentation/master/#as) which, unlike [`@intCast`](https://ziglang.org/documentation/master/#intCast),  has no bounds checking runtime overhead (in debug mode). That can then be demoted to just `1` if you prefer.